packages/fiori/src/ShellBar.ts

```diff
@@ -736,7 +798,7 @@ class ShellBar extends UI5Element {
 		});
 
 		for (let i = 0; i < itemsByPriority.length; i++) {
-			if (i < overflowCount) {
+			if (i < overflowCount.length) {
```

Comment:
This `overflowCount.length` also shows that the naming should be improved
`overflowCount` already looks like the number of items, but we call `overflowCount.length`

Better rename `overflowCount` to `overflowItems` or `overflowIcons` or whatever it is
and then in the loop this kind of expression `overflowItems.length` will look more logic